docs/maintainer-guide/releases.md

```diff
@@ -27,10 +31,10 @@ On the day of a scheduled release, the release team should follow these steps:
     * Small bug fixes written by a team member.
 1. Log into Jenkins and schedule a build for the "ESLint Release" job.
 1. Wait for the "ESLint Release" job to complete.
-1. Update the release blog post with a "Highlights", including new rules and anything else that's important.
-1. Make release announcement in the chatroom.
-1. Make release announcement on Twitter.
-1. Remind the team not to merge anything other than documentation changes and bug fixes.
+1. Update the release blog post with a "Highlights" section, including new rules and anything else that's important.
+1. Make a release announcement in the chatroom.
```

Comment:
Since you're doing a clarity pass, should we say "public chatroom" to be very clear here? Probably not necessary but something to consider.